### PR TITLE
[RTT-3700] Enable multi_org registration

### PR DESCRIPTION
**Pull request by marta-lewandowska**
Add support for user who is member of more than one organization to
be able to register and choose their organization from combobox.

### CODE REVIEW

File: anabot/runtime/installation/hub/connect_to_redhat/registration.py [hunk 1]
```diff
@@ -38,6 +38,12 @@ def registration_info_panel(local_node):
     sibling_filler = getparent(unreg_button, 'filler')
     return getsibling(sibling_filler, 1, "panel")
 
+handle_act('/account_organization', default_handler)
```
Comment (pholica) [inline anabot/runtime/installation/hub/connect_to_redhat/registration.py:41]: I believe that the "account_" can be dropped from the name as there should be no other "organization" related information in the information about the subscription/registration so the element path could simply be:
/installation/hub/connect_to_redhat/registration/organization

@jstodola what do you think?

File: anabot/runtime/installation/hub/connect_to_redhat/__init__.py [hunk 2]
```diff
@@ -300,16 +327,16 @@ def unregister_button(local_node, visible=True, sensitive=True):
     "Attaching subscription..."
 ]
 registered = [
-        "Registered.",
-        "Registered to Satellite."
+    "Registered.",
```
Comment (pholica) [inline anabot/runtime/installation/hub/connect_to_redhat/__init__.py:330]: Please don't do those formatting changes as part of one commit (together with implementing the organizations handling). Remove those changes or move them to a separate commit.

**Review by pholica — CHANGES_REQUESTED** [review]

File: anabot/runtime/installation/hub/connect_to_redhat/registration.py (hunk 1 above)
Comment (marta-lewandowska) [reply anabot/runtime/installation/hub/connect_to_redhat/registration.py:41]: @pholica I named it this way so that it would be consistent with the other functions that pertain to the combobox. But if you think it should be changed, I will change it.

File: anabot/runtime/installation/hub/connect_to_redhat/registration.py [hunk 3]
```diff
@@ -38,6 +38,12 @@ def registration_info_panel(local_node):
     sibling_filler = getparent(unreg_button, 'filler')
     return getsibling(sibling_filler, 1, "panel")
 
+handle_act('/organization', default_handler)
+@handle_chck('/account_organization')
```
Comment (pholica) [inline anabot/runtime/installation/hub/connect_to_redhat/registration.py:42]: The handler name doesn't correspond with the check name.

File: anabot/runtime/installation/hub/connect_to_redhat/__init__.py [hunk 4]
```diff
@@ -309,7 +336,7 @@ def unregister_button(local_node, visible=True, sensitive=True):
 def wait_until_registered_handler(element, app_node, local_node):
     register_button = getnode(local_node, "push button", tr("_Register", context="GUI|Subscription|Register"), sensitive=None, visible=None)
     status_label = getsibling(register_button, -1, "label", sensitive=None)
-    tr_register_phases = [ tr(phase) for phase in register_phases ]
+    tr_register_phases = [tr(phase) for phase in register_phases]
```
Comment (pholica) [inline anabot/runtime/installation/hub/connect_to_redhat/__init__.py:339]: Another formatting change that should be either removed or moved to separate commit.

File: anabot/runtime/installation/hub/connect_to_redhat/__init__.py [hunk 5]
```diff
@@ -152,6 +152,38 @@ def password_check(element, app_node, local_node):
...
+
+def account_organization_combo_check(element, app_node, local_node):
+    value = combo_str(get_attr(element, 'value'))
+    combobox = getnodes(local_node, 'combo box')[0]
+    if combobox.name != value:
+        return Fail('Different value was present: "%s". Expected: "%s".' % (combobox.name, value))
+    return PASS
+
+@handle_act('/account_organization')
+def account_organization_handler(element, app_node, local_node):
```
Comment (pholica) [inline anabot/runtime/installation/hub/connect_to_redhat/__init__.py:178]: I don't see a need for a definition of the `account_organization_handler` and `account_organization_check` functions as the only thing they do is calling `account_organization_combo_handler` and `account_organization_combo_check` respectively. The thing that can be done here is to not have the `account_organization_handler` and `account_organization_check` functions at all and just decorate the `account_organization_combo_handler` and `account_organization_combo_check` respectively.

**Review by pholica — APPROVED** [review]
